grpc/src/main/java/io/stargate/grpc/impl/GrpcImpl.java

```diff
@@ -39,6 +40,8 @@
 public class GrpcImpl {
   private static final Logger logger = LoggerFactory.getLogger(GrpcImpl.class);
   private static final Integer EXECUTOR_SIZE = Integer.getInteger("stargate.grpc.executor_size", 8);
+  private static final Boolean REFLECTION_ENABLED =
+      "true".equalsIgnoreCase(System.getProperty("stargate.grpc.reflection_enabled", "true"));
```

Comment:
In other places I think we simply use " Boolean.parseBoolean("...", "true");". It's bit inconsistent to allow case-insensitiveness here. Up to you if you want to leave it as-is.

Reply:
Ok, updated..